.github/workflows/docker_build.yml

```diff
@@ -1,21 +1,22 @@
-name: Build and push docker nightly
+name: Docker Build
```

Comment:
we might still want to keep the workflow, simply duplicate it. This is for the previous branch (DLC) to use the workflow

Reply:
Yeah. Another option which is more consistent but makes more changes is to cherry-pick this PR to the supported DLC branches. Then they will follow a similar process so we don't have to maintain both in docs. I'm fine with either if you have a strong preference